active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,225 @@
+}
+
+```
+
+## Theme Resources
+
+You can modify the look of a Breadcrumb by specifying Xaml resources in your app.
+For more info, see the
+[lightweight styling guide](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-styles#lightweight-styling).
+
+
+| Name | Description | 
+| :---------- | :------- | 
+| BreadcrumbNormalForegroundBrush | Sets the foreground color | 
+| BreadcrumbHoverForegroundBrush | Sets the hover foreground color |
+| BreadcrumbPressedForegroundBrush | Sets the pressed foreground color |
+| BreadcrumbDisabledForegroundBrush | Sets the disabled foreground color |
+| BreadcrumbFocusForegroundBrush | Sets the focus foreground color |
+| BreadcrumbCurrentNormalForegroundBrush | Sets the foreground color for the current item |
+| BreadcrumbCurrentHoverForegroundBrush | Sets the hover foreground color for the current item |
+| BreadcrumbCurrentPressedForegroundBrush | Sets the pressed foreground color for the current item |
+| BreadcrumbCurrentDisabledForegroundBrush | Sets the disabled foreground color for the current item |
+| BreadcrumbCurrentFocusForegroundBrush | Sets the focus foreground color for the current item |
+| BreadcrumbBackgroundBrush | Sets the background color |
+| BreadcrumbBorderBrush | Sets the border color |
+| BreadcrumbItemThemeFontSize | Sets the item font size | 
+
+
+# Appendix
+
+## Inputs and Accessibility 
+
+## UI Automation
+Breadcrumb is identified as a [UIA Navigation Landmark](https://docs.microsoft.com/windows/win32/winauto/landmark-type-identifiers). Each BreadcrumbItem will use a [InvokePattern](!https://docs.microsoft.com/dotnet/framework/ui-automation/implementing-the-ui-automation-invoke-control-pattern) with control type [Button](https://docs.microsoft.com/windows/win32/winauto/uiauto-supportbuttoncontroltype). 
+
+## Keyboard
+
+* The user can tab into the Breadcrumb and tab out.
+The first tab in gives focus to the first (highest-level) node (if the breadcrumb is crumbled, the ellipsis button instead).
+* Once the Breadcrumb has focus, the user can use the left/right arrows to navigate from node to node.
+* If the Breadcrumb is crumbled and focus is on the ellipsis
+  * invoking with space/enter key will open the flyout from the ellipsis, focus will move to the first item in the flyout list.
+  * The down/up keys will navigate between the flyout items. The ellipsis is not focusable while the flyout is open.
+  * From any item in the flyout the user can use right arrow to go to the node right of the ellipsis. 
+
+## Gamepad
+
+* Spatial navigation to focus and select the nodes. A flyout can be opened using the A button. 
+
+## Screen reader
+
+* When the focus is on a node, the screen reader will announce the node
+as well as "n of m" to convey which node in the path it is. 
+* If the Breadcrumb has been crumbled due to resizing
+  * the ellipsis is read as 'more' and is not counted for m (the total number of nodes)
```

Comment:
Just to see if I understand, if the breadcrumb has crumbled and looks like this:

[...] [A] [B]

then the screen reader will initially read "More. 0 of 2. Collapsed."

If you move to "A" then it will read "A. 1 of 2."

Reply:
@RBrid/Alberto to confirm. UIA counts are '1' based, so I think it would read 1/3, 2/3, 3/3. If the flyout opens, that will be a separate list and have 1/x, 2/x etc..

Reply:
Yes, as @ranjeshj  mentioned "A" and "B" are 1 indexed out of the visible items, while the items in the drop-down are indexed in their context. The ellipsis should not be enumerated

Reply:
Recommend: Update description